moneta-core/src/main/java/org/javamoney/moneta/internal/format/DefaultMonetaryAmountFormat.java

```diff
@@ -105,7 +105,7 @@ private void initPattern(String pattern, List<FormatToken> tokens,
         }
     }
 
-    private boolean isLiteralPattern(String pattern, AmountFormatContext style) {
+    private boolean isLiteralPattern(String pattern) {
```

Comment:
maybe this was unfinished refactoring

Reply:
Even if it was. Is master the place for unfinished refactorings?